rc-events-service/src/main/scala/com/epam/random_coffee/events/model/Event.scala

```diff
@@ -1,5 +1,20 @@
 package com.epam.random_coffee.events.model
 
-final case class Event(id: EventId, name: String)
+import java.time.Instant
 
 final case class EventId(value: String) extends AnyVal
+
+final case class Event(id: EventId, name: String)
+
+final case class Author(id: String) extends AnyVal
+
+final case class DateForEvent(date: Instant) extends AnyVal
+
+final case class RandomCoffeeEvent(
```

Comment:
What the differences between RandomCoffeeEvent and Event?
Please, change naming for that class(es)

Reply:
Event is a dummy class and will be fully replaced later by RandomCoffeeEvent. We use Event for "delete" and "update" methods now.

Reply:
Ok, I replaced Event with DummyEvent to show that it's just a temporary feature

Reply:
Why not use `RandomCoffeeEvent` everywhere?

Reply:
Done, now we have only RandomCoffeeEvent